src/test/java/de/tum/in/www1/artemis/assessment/GradingScaleUtilService.java

```diff
@@ -110,6 +140,15 @@ public GradingScale generateGradingScale(int gradeStepCount, double[] intervals,
         return gradingScale;
     }
 
+    /**
+     * Creates a grading scale with a sticky grade step.
```

Comment:
Here you could also consider explaining the fail statement, or at least mention that there is one.